testsuite/integration-tests-spring/deployment/src/test/java/org/jboss/resteasy/test/spring/deployment/AddSpringResteasyAsResourceRootTest.java

```diff
@@ -77,6 +82,16 @@ private static WebArchive createDeploymentWithSpringMvcDispatcher() {
                 .addAsWebInfResource(AddSpringResteasyAsResourceRootTest.class.getPackage(), "mvc-dispatcher-servlet/web.xml", "web.xml")
                 .addAsWebInfResource(AddSpringResteasyAsResourceRootTest.class.getPackage(), "mvc-dispatcher-servlet/mvc-dispatcher-servlet.xml", "mvc-dispatcher-servlet.xml")
                 .addAsWebInfResource(AddSpringResteasyAsResourceRootTest.class.getPackage(), "mvc-dispatcher-servlet/applicationContext.xml", "applicationContext.xml");
+
+        archive.addAsManifestResource(PermissionUtil.createPermissionsXmlAsset(
+            new PropertyPermission("arquillian.*", "read"),
+            new ReflectPermission("suppressAccessChecks"),
+            new RuntimePermission("accessDeclaredMembers"),
+            new RuntimePermission("getClassLoader"),
+            new FilePermission("<<ALL FILES>>", "read"),
+            new LoggingPermission("control", "")
+        ), "permissions.xml");
```

Comment:
I get the failure intermittently. It is reproducible when I comment out deployment dep1 and dep2 and run the AddSpringResteasyAsResourceRootTest. First time it fails, subsequent runs pass.

Independently on this intermittent issue, i see these info messages from spring in the log:

> 17:40:30,618 INFO  [org.springframework.web.context.support.StandardServletEnvironment] (ServerService Thread Pool -- 19) Caught AccessControlException when accessing system property [spring.liveBeansView.mbeanDomain]; its value will be returned [null]. Reason: WFSM000001: Permission check failed (permission "("java.util.PropertyPermission" "spring.liveBeansView.mbeanDomain" "read")" in code source "(vfs:/content/deploymentWithSpringMvcDispatcherSpringIncluded.war/WEB-INF/lib/spring-core-4.2.4.RELEASE.jar <no signer certificates>)" of "null")

But the test passes even with them.